### ISSUE TITLE
Something really wrong with version .17

### ISSUE THREAD

**Issue by nvitas** [taniman/profit-trailer#1144]
Upgraded this morning because I was having the re-buy issue where it would buy a pair right after it would sell it, almost instantly.

Saw that v.17 fixes that so I upgraded then I started seeing this stuff in the logs:

2018-02-17 12:50:10 INFO DCAStrategyRunner - There is not enough volume to make a successfull DCA OMGETH
2018-02-17 12:59:38 INFO ProfitTrailerService - Cannot buy OMGETH. rebuy timeout active!
2018-02-17 13:02:04 INFO ProfitTrailerService - Cannot buy LSKETH. rebuy timeout active!
2018-02-17 13:03:25 INFO DCAStrategyRunner - There is not enough volume to make a successfull DCA LSKETH

---this goes on for a couple of hours before i caught it ------

2018-02-17 14:21:03 INFO ProfitTrailerService - Cannot buy OMGETH. rebuy timeout active!
2018-02-17 14:21:21 INFO ProfitTrailerService - Cannot buy LSKETH. rebuy timeout active!
2018-02-17 14:21:44 INFO ProfitTrailerService - Normal Heartbeat
2018-02-17 14:22:04 INFO DCAStrategyRunner - There is not enough volume to make a successfull DCA OMGETH
2018-02-17 14:22:26 INFO ProfitTrailerService - DCA Heartbeat
2018-02-17 14:22:27 INFO DCAStrategyRunner - There is not enough volume to make a successfull DCA LSKETH

Finally restarted and now I get this:

INFO: Initializing Spring FrameworkServlet 'dispatcherServlet'
2018-02-17 14:23:39 ERROR ProfitTrailerService - Something is really wrong
2018-02-17 14:23:39 INFO ProfitTrailerService - Seems like the exchange is spitt
ing out invalid data. Let's wait 5 minutes and see

I restarted again and it was up and running normally but now am getting this for a pair that it just bought

2018-02-17 14:34:25 INFO NormalStrategyRunner - There is not enough volume to buy APPCETH

Once those errors started showing up the price wasn't updating any more and the bot was frozen.

Basically once it buys a pair DCA no longer works because of the rebuy timeout active & not enough volume messages.

**Comment by SgtBatten** [comment]
Did you set the rebuy parameters? They are new in 17 and you have to add them. Even if you want 0

**Comment by acefed** [comment]
...and be sure to restart, just setting the rebuy parameters and letting the bot (auto)reload the new configs didn't work for me.

**Comment by minerdude** [comment]
I'm.getting theses messages too in DCA, my rebuy timeout is 0 both in pairs and em dca

**Comment by SgtBatten** [comment]
It's fixed already in internal testing. Rollback for now until 18 is realeased soonish
